chapters/equations.tex

```diff
@@ -887,4 +887,51 @@ \subsection{Recommended selection of start values}\label{recommended-selection-o
 The model component hierarchy level number is used to give \lstinline!start!-attribute a confidence number, where a lower number means that the \lstinline!start!-attribute is more confident.
 Loosely, if the \lstinline!start!-attribute is set or modified on level $i$ then the confidence number is $i$.
 If a \lstinline!start!-attribute is set by a possibly hierarchical modifier at the top level, then this \lstinline!start!-attribute has the highest confidence, namely 1 irrespectively on what level, the variable itself is declared.
+If the \lstinline!start!-attribute is set equal to parameter, which may be equal which may be equal to another parameter (etc), the lowest confidence number of these bindings are used.
```

Comment:
```suggestion
If the \lstinline!start!-attribute is set equal to a parameter, which may be equal which may be equal to another parameter (etc), the lowest confidence number of these bindings are used.
```

Reply:
"which may be equal which may be equal" is definitely redundant, both in the original and changed versions. Once it has been agreed whether or not to accept this change, I'd suggest to remove one "which may be equal" from the sentence.